src/studies/experiment_postdictive.jl

```diff
@@ -0,0 +1,246 @@
+    original_stdout = stdout
+    original_stderr = stderr
+    file_log = open("log.txt", "w")
+
+    try
+        redirect_stdout(file_log)
+        redirect_stderr(file_log)
+
+        # Get default parameters for the study
+        _, act = study_parameters(:Postdictive)
+
+        run_postdictive_case(device, shot; sty.fit_profiles, sty.use_local_cache, sty.save_folder)
+
+        # catch error
+        #     if isa(error, InterruptException)
+        #         rethrow(error)
+        #     end
+        #     @error "Error in postdictive case for $(device) shot $(shot): $error"
+    finally
+        redirect_stdout(original_stdout)
+        redirect_stderr(original_stderr)
+        cd(original_dir)
+        close(file_log)
+    end
+end
+
+function run_postdictive_case(device::Symbol, shot::Int; kw...)
+    dd = IMAS.dd()
+    dd_exp = IMAS.dd()
+    run_postdictive_case!(dd, dd_exp, device, shot; kw...)
+    return (dd=dd, dd_exp=dd_exp)
+end
+
+function run_postdictive_case!(dd::IMAS.dd, dd_exp::IMAS.dd, device::Symbol, shot::Int; fit_profiles::Bool, use_local_cache::Bool, save_folder::AbstractString=abspath("."))
+    savedir = abspath(joinpath(save_folder, "$(device)_$(shot)__$(Dates.now())__$(getpid())"))
+    if !isdir(savedir)
+        mkdir(savedir)
+    end
+
+    # Get case parameters
+    ini, act = FUSE.case_parameters(device, shot; fit_profiles, use_local_cache)
+
+    # Override act with study-specific actor parameters
+    #merge!(act, act)
+
+    # init
+    ini.time.simulation_start = ini.general.dd.equilibrium.time_slice[2].time
+    FUSE.init!(dd, ini, act)
+
+    # keep aside the dd with experimental data
+    IMAS.fill!(dd_exp, dd)
+
+    # identify LH transitions
+    experiment_LH = FUSE.LH_analysis(dd; do_plot=false)
+
+    act.ActorPedestal.model = :dynamic
+    act.ActorPedestal.tau_n = experiment_LH.tau_n
+    act.ActorPedestal.tau_t = experiment_LH.tau_t
+    act.ActorWPED.ped_to_core_fraction = experiment_LH.W_ped_to_core_fraction
+    act.ActorEPED.ped_factor = 1.0
+    act.ActorPedestal.T_ratio_pedestal = 1.0 # Ti/Te in the pedestal
+
+    if true
+        # density and Zeff from experiment
+        act.ActorPedestal.density_ratio_L_over_H = 1.0
+        act.ActorPedestal.zeff_ratio_L_over_H = 1.0
+    else
+        # density can go from L to H mode at a different time
+        act.ActorPedestal.density_ratio_L_over_H = experiment_LH.ne_L_over_H
+        act.ActorPedestal.zeff_ratio_L_over_H = experiment_LH.zeff_L_over_H
+        dd.pulse_schedule.density_control.n_e_line.reference = experiment_LH.ne_H
+        dd.pulse_schedule.density_control.zeff_pedestal.reference = experiment_LH.zeff_H
+    end
+
+    if false
+        # LH-transition from LH scaling law
+        act.ActorPedestal.mode_transitions = missing
+    else
+        # LH-transition at user-defined times
+        act.ActorPedestal.mode_transitions = experiment_LH.mode_transitions
+        act.ActorPedestal.mode_transitions[5.2] = :L_mode
+    end
```

Comment:
Are you sure you want to keep this hardcoded? A parameter in sty could handle this for you